homeassistant/const.py

```diff
@@ -524,16 +524,37 @@ class Platform(StrEnum):
 TIME_MONTHS: Final = "m"
 TIME_YEARS: Final = "y"
 
+
 # Length units
+class UnitOfLength(StrEnum):
```

Comment:
Just wondering: should we keep `..Of..`? I think `UnitLength` could be enough, especially if a specific one is referenced like `UnitLength.METERS`.

Reply:
My original proposal was `LengthUnit` but then you loose the grouping in the imports (`UnitOfXXX`).

IMO, `UnitLength` isn't really English, but it does make it a little shorter...

I'm not 100% sure TBH

Reply:
<https://en.wikipedia.org/wiki/Unit_of_length> 😉